src/client.ts

```diff
@@ -230,10 +230,25 @@ export class Turbopuffer {
       region,
       defaultNamespace,
       ...opts,
-      baseURL: baseURL || `https://${region}.turbopuffer.com`,
+      baseURL,
     };
 
-    this.baseURL = options.baseURL!;
+    baseURL = baseURL || 'https://{region}.turbopuffer.com';
```

Comment:
The best UX here would be that `region` can simply start with `https` probably, and kill `baseURL` entirely...

Reply:
Discussed with @sirupsen on Slack and got the 👍🏽 on this for now.